test/core/keyTests.ts

```diff
@@ -0,0 +1,45 @@
+﻿///<reference path="../testReference.ts" />
+
+describe("DatasetKey", () => {
+  it("is passed to Dataset", () => {
+    let ds: Plottable.Dataset = new Plottable.Dataset();
+    let key = (d: any, i: number) => { return i; };
+    ds.keyFunction(key);
+    assert.deepEqual(ds.keyFunction(), key, "key is passed to dataset");
+  });
+
+  it("may accept noConstancy predefined key function", () => {
+    let ds: Plottable.Dataset = new Plottable.Dataset();
+    ds.keyFunction(Plottable.KeyFunctions.noConstancy);
+    let d = { foo: "bar" };
+    let a = ds.keyFunction()(d, 1);
+    let b = ds.keyFunction()(d, 1);
+    assert.isTrue(b - a === 1, "invocations give numerically increasing results");
```

Comment:
For this assertion, can we change it to
`assert.strictEqual(b, a + 1...)`?

`assert.strictEqual` is significantly more useful than `assert.isTrue`.  Please apply this idea to other occurances